db/seeds.rb

```diff
@@ -1 +1,26 @@
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+if Rails.env.development?
```

Comment:
Should we move the Units to outside the development env? As we'll need them in prod and staging as well.

Reply:
Right! Also, since we can edit them in the interface I'll add a condition to only set units from seeds for new records.